### PR TITLE
Receive glucose data from Syai Tag App.

### PR DESCRIPTION
**Pull request by syai-dev**
Add blood glucose data source.

Screenshot_20240820-175444_AAPS

### CODE REVIEW

File: database/impl/src/main/kotlin/app/aaps/database/entities/GlucoseValue.kt [hunk 1]
```diff
@@ -103,6 +103,7 @@ data class GlucoseValue(
         AIDEX,
         RANDOM,
         UNKNOWN,
+        SyaiTag,
```
Comment (Philoul) [inline database/impl/src/main/kotlin/app/aaps/database/entities/GlucoseValue.kt:106]: I think SYAITAG should be put in uppercase to be consistant with the other items of the list

**Comment by Philoul** [comment]
Please move Syai Tag plugin before Random BG plugin in the list (Random BG should always be the latest BG source because not usable for loop).

**Comment by MilosKozak** [comment]
@syai-dev did you create PR for wiki?

**Comment by syai-dev** [comment]
> @syai-dev did you create PR for wiki?

NO

File: database/impl/src/main/kotlin/app/aaps/database/entities/GlucoseValue.kt (hunk 1 above)
Comment (syai-dev) [reply database/impl/src/main/kotlin/app/aaps/database/entities/GlucoseValue.kt]: We have updated

**Comment by syai-dev** [comment]
we have already create PR for wiki https://github.com/openaps/AndroidAPSdocs/pull/2069

**Comment by MilosKozak** [comment]
great. is the CGM available somewhere in europe to be able test it?

**Comment by syai-dev** [comment]
> great. is the CGM available somewhere in europe to be able test it?

yes

**Comment by MilosKozak** [comment]
are you able to give me a contact?

**Comment by syai-dev** [comment]
me? Or a tester?

**Comment by syai-dev** [comment]
> are you able to give me a contact?

Sure, apologies for the delayed response as we were implementing hardware updates to the CGM; production and tests took a while. 
Please contact us via [derick.liu@syai.com](mailto:derick.liu@syai.com) so we can see if a delivery can be arranged for you. We appreciate the follow-up.

**Pull request merged**